### PR TITLE
Upgrade to the 5.0 stack and add more kibana shots

### PR DESCRIPTION
**Pull request by jamiesmith**
Manually merged the stuff I changed.  Sometimes git sucks.

@petemiron please have a look. Also, @dnblankedelman if you have a chance.  Thanks!

### CODE REVIEW

File: example-elk-stack/README.md [hunk 1]
```diff
@@ -832,32 +945,33 @@ We also include an application manifest for our syslogger
 
 ```
 
+> Note that you must replace the x.x.x.x with the IP address of your TCP router.
```
Comment (petemiron) [inline example-elk-stack/README.md]: This is confusing to me. Which IP addresses are we supposed to use? how are we supposed to find them? I think I've set everything up correctly (even have drains on both public IP addresses of demo.apcera.net) and I can't seem to get logstash to receive syslog messages and work.

File: example-elk-stack/README.md [hunk 2]
```diff
@@ -47,144 +61,202 @@ directory tree so you can see how we stored the pieces in the filesystem.  In
...
+the x-pack plugin.  Note that the `echo "y"` in the script is accepting the
+trial license.  Finally, it changes the ownership of the whole tree to `runner`,
+a predefined user, and sets up a soft link to make the paths nicer.
 
 The `environment` section adds `/opt/apcera/elasticsearch/bin` to the path.
 When this package is included in a container, the environment will be updated
 to include this.
 
-If you don't want to include shield, simple omit the two `plugin install` lines.
+If you don't want to include x-pack, simple omit the two `plugin install` lines.
```
Comment (jamiesmith) [inline example-elk-stack/README.md]: fixed pending checkin

File: example-elk-stack/README.md (hunk 1 above)
Comment (jamiesmith) [inline example-elk-stack/README.md]: replaced with auto - much cleaner now.  pending checkin

File: example-elk-stack/README.md [hunk 3]
```diff
@@ -1150,6 +1150,12 @@ For more information on filters and related topics on logstash, take a look at:
 * [filters](http://tinyurl.com/nlwwjnr)
 * [codecs](http://tinyurl.com/p8tzgtc)
 
+Also new to the ELK stack is [Beats](https://www.elastic.co/products/beats), 
+Elastic's "platform for single-purpose data shippers". Beats are lightweight
```
Comment (petemiron) [inline example-elk-stack/README.md]: word missing? lightweight ...

**Pull request merged**